Mathlib/Data/DList/Basic.lean

```diff
@@ -0,0 +1,45 @@
+/-
+Copyright (c) 2018 Simon Hudon. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Simon Hudon
+-/
+import Std.Data.DList
+import Mathlib.Mathport.Rename
+
+
+/-!
+# Difference list
+
+This file provides a few results about `DList`, which is defined in `Std`.
+
+A difference list is a function that, given a list, returns the original content of the
+difference list prepended to the given list. It is useful to represent elements of a given type
+as `a₁ + ... + aₙ` where `+ : α → α → α` is any operation, without actually computing.
+
+This structure supports `O(1)` `append` and `concat` operations on lists, making it
+useful for append-heavy uses such as logging and pretty printing.
+-/
+
+
```

Comment:
How about `open Std` here?

Reply:
Actually it should be `namespace Std` since e.g. `DList.join` needs to be `Std.DList.join` since it's about `Std.DList`.

Reply:
@arienmalec, it's best if you click "Resolve conversation" yourself when you've made the appropriate change. It makes life easier when there are multiple reviewers, and we trust you to get it right. :-)

Reply:
You should also remove the `awaiting-author` tag and set it back to `awaiting-review`, or it might get stuck in limbo!